pkg/orchestration/wiring/wiringutil/knownshapes/standard_test.go

```diff
@@ -24,7 +24,7 @@ func TestAllKnownShapes(t *testing.T) {
 	t.Parallel()
 
 	allKnownShapes := []wiringplugin.Shape{
-		NewBindableEnvironmentVariables(resourceName),
+		NewBindableEnvironmentVariables(resourceName, "abc", map[string]string{"a": "b"}),
```

Comment:
This is still not catching the broken deep copy - we forgot to run generator and don't have CI setup on this repo properly. It does not ensure generator was run.